react-effects/src/index.jsx

```diff
@@ -9,18 +9,18 @@ function List() {
 
   // Your code here:
   //  - Read the items using `readItems` and update state so the list displays
-  const promise = readItems();
   useEffect(() => {
-    if (isLoading) {
-      setIsLoading(!isLoading);
-    }
+    const promise = readItems();
     promise.then((value) => {
+      if (isLoading) {
```

Comment:
So you don't really need to check `isLoading` here, do you? You know that once `readItems` completes then you will no longer be loading.